### PR TITLE
Make tooltip orange when on backuplist

### PR DESCRIPTION
**Pull request by ysbrandB**
Change the tooltip on the included events-block from green to orange when on the backup list and announce that you are on the backuplist with the tooltip. Avoids confusion.

### CODE REVIEW

**Review by Jessseee — CHANGES_REQUESTED** [review]
I think a helper method in the activity model, like isParticipating but for the backuplist, would better than doing database queries in the blade file.

**Comment by ysbrandB** [comment]
> I think a helper method in the activity model, like isParticipating but for the backuplist, would better than doing database queries in the blade file.

The logic is now moved to the controller

File: resources/views/event/display_includes/event_block.blade.php [hunk 1]
```diff
@@ -28,10 +28,13 @@
                    data-toggle="tooltip" data-placement="top" title="This is a featured activity!"></i>
             @endif
             @if($event->activity && Auth::check())
-                @if($event->activity->isParticipating(Auth::user()))
-                    <i class="fas fa-check text-primary fa-fw" aria-hidden="true"
-                       data-toggle="tooltip" data-placement="top" title="You are participating!"></i>
-                @endif
```
Comment (Jessseee) [inline resources/views/event/display_includes/event_block.blade.php:34]: This will now always show the user as participating, right?

**Review by Jessseee — CHANGES_REQUESTED** [review]

**Review by Jessseee — CHANGES_REQUESTED** [review]

**Review by Jessseee — APPROVED** [review]
LGTM

**Pull request merged**